cloud/cloud.go

```diff
@@ -127,15 +127,16 @@ func syncFiles(username string, hostname string, box *devbox.Devbox) error {
 	return nil
 }
 
-func shell(username string, hostname string) error {
+func shell(username string, hostname string, box *devbox.Devbox) error {
```

Comment:
I don't love passing box as an argument here, but it's fine for a quick implementation.

Reply:
what is your concern?

For now, we use `box.ConfigDir()` so another alternative is to pass the `configDir string` as an argument.